host_core/native/hostcore_wasmcloud_native/src/wasmruntime.rs

```diff
@@ -103,10 +154,22 @@ pub fn on_load(env: Env) -> bool {
 #[rustler::nif(name = "runtime_new")]
 pub fn new<'a>(
     env: rustler::Env<'a>,
-    _config: ExRuntimeConfig,
+    ExRuntimeConfig { host_id }: ExRuntimeConfig,
 ) -> Result<ResourceArc<RuntimeResource>, rustler::Error> {
-    let host_handler = ElixirHandler { pid: env.pid() };
-    let rt = WcRuntime::from_host_handler(host_handler)
+    tracing_subscriber::registry()
+        .with(tracing_subscriber::fmt::layer().without_time())
```

Comment:
we can't use a tracing subscriber inside a nif

Reply:
Why not?